lightning/src/ln/channelmonitor.rs

```diff
@@ -383,38 +382,45 @@ struct LocalSignedTx {
 	htlc_outputs: Vec<(HTLCOutputInCommitment, Option<Signature>, Option<HTLCSource>)>,
 }
 
-#[derive(PartialEq)]
-enum InputDescriptors {
-	RevokedOfferedHTLC,
-	RevokedReceivedHTLC,
-	OfferedHTLC,
-	ReceivedHTLC,
-	RevokedOutput, // either a revoked to_local output on commitment tx, a revoked HTLC-Timeout output or a revoked HTLC-Success output
-}
-
 /// When ChannelMonitor discovers an onchain outpoint being a step of a channel and that it needs
 /// to generate a tx to push channel state forward, we cache outpoint-solving tx material to build
 /// a new bumped one in case of lenghty confirmation delay
 #[derive(Clone, PartialEq)]
-enum InputMaterial {
+pub(crate) enum InputMaterial {
 	Revoked {
+		/// Witness script
```

Comment:
Note that the documentation-required checks don't apply to pub(crate) things. Two word comments probably indicate you should just change the variable name and move on :)